tests/integration/test_storage_kerberized_hdfs/hdfs_configs/bootstrap.sh

```diff
@@ -111,6 +111,23 @@ cat > /usr/local/hadoop/etc/hadoop/hdfs-site.xml << EOF
   <name>dfs.datanode.http.address</name>
   <value>0.0.0.0:1006</value>
 </property>
+<!-- If the port is 0 then the server will start on a free port. -->
```

Comment:
I see the linked issue https://github.com/ClickHouse/ClickHouse/issues/48975 is about `test_storage_hdfs`, but this file is related to `test_storage_kerberized_hdfs` only, so it does not fix the issue?

Reply:
@nickitat let's just copy this config change into all HDFS tests?